satpy/readers/hrit_base.py

```diff
@@ -222,6 +221,16 @@ def _get_hd(self, hdr_info):
                                              'SSP_longitude': 0.0}
         self.mda['orbital_parameters'] = {}
 
+    @property
+    def observation_start_time(self):
+        """Get start time."""
+        return self._start_time
```

Comment:
This is because if I recall correctly the test for seviri_l1b_hrit.py was failing because there observation start/end time was not created in the hrit_base. Hence the addition here. 
But AFAIK hrit_base is used for other reader  of other instrument that does not necessarily follow the same observation/nominal rule as SEVIRI. 

Hence I decided to only replicate the existing start/end time and not meddle with it too much as IMHO this need to.be conducted in a dedicated issue/PR.

Reply:
see 
https://github.com/pytroll/satpy/pull/2420/commits/46eb8f0f2ac8e0f5d62563775da1b2f118d6fb9e
and 
https://github.com/pytroll/satpy/pull/2420/commits/d77d2f2ac49cd01f2df2247b1afcaf1bc6b9a691